contribute.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Phabricator",
+  "description": "Mozilla's Phabricator instance for code review.",
+  "repository": {
+    "url": "https://github.com/mozilla-services/mozphab",
+    "license": "MPL2",
+    "tests": "https://circleci.com/gh/mozilla-services/mozphab/tree/master"
+  },
+  "participate": {
+    "home": "https://moz-conduit.readthedocs.io/en/latest/",
+    "docs": "https://moz-conduit.readthedocs.io/en/latest/",
+    "mailing-list": "https://www.mozilla.org/en-US/about/forums/#tools",
+    "irc": "irc://irc.mozilla.org/#phabricator",
+    "irc-contacts": [
+      "mcote",
+      "dkl",
```

Comment:
Hehe not sure how dkl became the lucky one here. :) To be fair I think maybe just ckolos and I should be listed.

Reply:
I shouldn't be in here at all tbh.